src/components/_forms/BondForm/index.tsx

```diff
@@ -152,39 +153,75 @@ export const BondForm: VFC<BondFormProps> = ({ onClose }) => {
         align="stretch"
         onSubmit={handleSubmit(onSubmit, onError)}
       >
-        <VStack align="stretch">
-          <CardHeading>Bonding Period</CardHeading>
-          <BondingPeriodOptions cellarConfig={cellarConfig} />
-        </VStack>
         <FormControl isInvalid={isError as boolean | undefined}>
-          <InputGroup display="flex" alignItems="center">
-            <ModalInput
-              type="number"
-              step="any"
-              {...register("depositAmount", {
-                required: "Enter amount",
-                valueAsNumber: true,
-                validate: {
-                  positive: (v) =>
-                    v > 0 || "You must submit a positive amount.",
-                  balance: (v) =>
-                    v <=
-                      parseFloat(
-                        toEther(lpTokenData?.formatted, 18, false)
-                      ) || "Insufficient balance",
-                },
-              })}
-            />
-            <InputRightElement h="100%" mr={3}>
-              <SecondaryButton
-                size="sm"
-                borderRadius={8}
-                onClick={setMax}
-              >
-                Max
-              </SecondaryButton>
-            </InputRightElement>
-          </InputGroup>
+          <HStack
+            p={4}
+            justifyContent="space-between"
+            w="100%"
+            bg="surface.secondary"
+            border="none"
+            borderRadius={16}
+            appearance="none"
+            textAlign="start"
+            css={{
+              "span:first-of-type": {
+                display: "flex",
+                justifyContent: "space-between",
+                alignItems: "center",
+              },
+            }}
```

Comment:
I'm pretty sure you can move these styles into the `_first` prop of the `<Text as="span" ... />` component on line 206!

Reply:
Thanks Xan, I'll implement this